### ISSUE TITLE
experiments reproducibility issue

### ISSUE THREAD

**Issue by DingchenYang99** [HQHBench/HQHBench#1]
hi,
this is an awesome work!
I have been running llava-v1.5-7b and instructblip-vicuna-7b locally on my device, and get the following experimental results:

[llava1.5]: 

greedy: {'overall': 0.375, 'action': 0.295, 'attribute': 0.16, 'comparison': 0.36, 'count': 0.345, 'environment': 0.375, 'existence': 0.645, 'OCR': 0.36, 'relation': 0.46}

sample with p=0.9: {'overall': 0.48125, 'action': 0.405, 'attribute': 0.22, 'comparison': 0.47, 'count': 0.49, 'environment': 0.425, 'existence': 0.695, 'OCR': 0.53, 'relation': 0.615}

beamsearch3: {'overall': 0.37875, 'action': 0.375, 'attribute': 0.17, 'comparison': 0.34, 'count': 0.43, 'environment': 0.35, 'existence': 0.585, 'OCR': 0.33, 'relation': 0.45}

[instructblip]

greedy : {'overall': 0.470625, 'action': 0.33, 'attribute': 0.225, 'comparison': 0.595, 'count': 0.295, 'environment': 0.52, 'existence': 0.56, 'OCR': 0.645, 'relation': 0.595}

sample with p=0.9 : {'overall': 0.5891181988742964, 'action': 0.495, 'attribute': 0.335, 'comparison': 0.7, 'count': 0.395, 'environment': 0.665, 'existence': 0.645, 'OCR': 0.7638190954773869, 'relation': 0.715}

These results show some disparity with the results you've reported in table 5.

Would you please teach me about where I've got it wrong. I followed evaluate.py, where model="gpt-3.5-turbo".

I would be happy to provide you with my evaluation results (I write them in a jsonl file.)

Thanks!🙏

**Comment by HQHBench** [comment]
Hi, 

Thanks for your attention to our work.

For efficiency reasons during large-scale evaluation, we utilize GPT to assess the responses of all models simultaneously for each image-question pair, which differs slightly from the code we provided and may have led to some deviation in reproducibility results. We have updated the prompts used in the actual large-scale evaluation and provided a specific example of the evaluation.
